.github/workflows/alpha_release.yml

```diff
@@ -36,6 +36,20 @@ jobs:
       with: 
         files: ./src/consts.ts
         replacements: '<INSTRUMENTATION_KEY>=${{ secrets.INSTRUMENTATION_KEY }}'
+    - name: Get Chat Version
```

Comment:
for a later stage, but i think you can define it as a composite action and reuse it 
https://docs.github.com/en/actions/creating-actions/creating-a-composite-action